### ISSUE TITLE
MD034 no bare url should be ignored in quote

### ISSUE THREAD

**Issue by simkimsia** [markdownlint/markdownlint#253]
Similar to #191 but for quote segments.

e.g. 

```
>Thanks for your input! I'd like to say please go to http://example.com or http://example2.com
```

The first one gets raised. I rather all are not raised since inside a quote.
